src/apps/explorer/pages/TransactionDetails.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import { useParams } from 'react-router'
+
+import { isAddress } from 'web3-utils'
+
+import NotFound from './NotFound'
+import OrdersTableWidget from '../components/OrdersTableWidget'
+import { useNetworkId } from 'state/network'
+import { BlockExplorerLink } from 'components/common/BlockExplorerLink'
+import { TitleAddress, Wrapper } from 'apps/explorer/pages/styled'
+
+const TransactionDetails: React.FC = () => {
+  const { transactionId } = useParams<{ transactionId: string }>()
+  const networkId = useNetworkId() || undefined
+  if (!isAddress(transactionId)) {
+    return <NotFound />
+  } else {
+    return (
+      <Wrapper>
+        <h1>
+          Transaction details
+          <TitleAddress
+            textToCopy={transactionId}
+            contentsToDisplay={<BlockExplorerLink type="address" networkId={networkId} identifier={transactionId} />}
+          />
+        </h1>
+        <OrdersTableWidget ownerAddress={transactionId} networkId={networkId} />
```

Comment:
While this does make sense, since it's almost 100% the same thing (not from design point of view, what Agustin proposed is the same, but looking at it now I can think of a few changes), this is creating =P

But seriously though, the widget doesn't apply here as tx logic is different than address logic.

We could potentially refactor it since they might be quite similar.

Anyway, this can be handled in a follow up PR.
For the first one I wouldn't expect to have this change at all, but I'm ok leaving it here for now.